man/coap_resource.txt.in

```diff
@@ -133,6 +133,13 @@ Set the notification message type to confirmable for any trigggered
 *COAP_RESOURCE_FLAGS_RELEASE_URI*::
 Free off the coap_str_const_t for _uri_path_ when the _resource_ is deleted.
 
+*COAP_RESOURCE_FLAGS_FORCE_SINGLE_BODY*::
+Force all large traffic to this resource to be presented as a single body
+to the request handler.
+
+*COAP_RESOURCE_FLAGS_OSCORE_ONLY*::
```

Comment:
This is in place to support Web linking as per [RFC8613 9 Web linking](https://www.rfc-editor.org/rfc/rfc8613#section-9) so that osc is returned in GET /.well-known/core and I think should be left as is.  If changed, the interop tests may fail.

I do like the concept of COAP_RESOURCE_FLAGS_SECURE_ACCESS_ONLY - suggest as a different PR, but have no real idea how/what to return in the .well-known/core information.